.github/workflows/trigger-load-tests.yml

```diff
@@ -11,53 +11,15 @@ on:
         description: 'Cluster name'
         default: 'oracledb-perf-cluster-test'
         required: false
+      branch:
+        description: 'Branch of the given repository'
+        default: ''
+        required: false
   schedule:
     - cron: '30 22 * * *'
 
 jobs:
-  invokeLoadTests:
-    runs-on: ubuntu-latest
-    if: github.repository_owner == 'ballerina-platform'
-    steps:
-      - name: Checkout
-        uses: actions/checkout@v2
-      - name: Process cluster name
-        id: processClusterName
-        run: |
-          clusterName="${{github.event.inputs.clusterName}}"
-          if [ "$clusterName" = "" ]; then
-            clusterName="oracledb-perf-cluster-test"
-          fi
-          printf "clusterName=%s" $clusterName >> $GITHUB_OUTPUT
-      - name: Get latest ballerina runtime artifact
-        id: setRuntimeUrl
-        shell: bash
-        run: |
-          printf "runtimeUrl=%s" $(curl -H "Accept: application/vnd.github.v3+json" \
-            https://api.github.com/repos/ballerina-platform/module-ballerinax-oracledb/actions/artifacts | \
-            jq '[.artifacts[] | select(.name=="ballerina-runtime" and .expired==false)]|max|.archive_download_url') >> $GITHUB_OUTPUT
-      - name: Format test names
-        id: formatTestNames
-        shell: bash
-        run: |
-          tests="${{github.event.inputs.tests}}"
-          if [ ! "$tests" = "" ]; then
-            testsInput=(${tests//,/ })
-            testsCount=${#testsInput[@]}
-            testsOutput=""
-            for (( i=0; i<$testsCount; i++ ));
-            do
-                testsOutput+=\\'"'${testsInput[$i]}\\'"',
-            done
-            printf "testNames=%s" [${testsOutput:0:-1}] >> $GITHUB_OUTPUT
-          else
-            printf "testNames=%s" $tests >> $GITHUB_OUTPUT
-          fi
-      - name: Invoke standard library load test workflow
-        uses: benc-uk/workflow-dispatch@v1
-        with:
-          workflow: Stdlib Workflow
-          repo: ballerina-platform/ballerina-performance-cloud
-          ref: refs/heads/main
-          token: ${{ secrets.BALLERINA_BOT_TOKEN }}
-          inputs: '{ "repo-name": "module-ballerinax-oracledb", "tests": "${{ steps.formatTestNames.outputs.testNames }}", "zipURL": ${{ steps.setRuntimeUrl.outputs.runtimeUrl}}, "clusterName": "${{ steps.processClusterName.outputs.clusterName }}", "dispatchType": "oracledb-load-test" }'
+  call_stdlib_trigger_load_test_workflow:
+    name: Run StdLib Load Test Workflow
+    if: ${{ github.event_name != 'schedule' || (github.event_name == 'schedule' && github.repository_owner == 'ballerina-platform') }}
+    uses: ballerina-platform/ballerina-standard-library/.github/workflows/trigger-load-tests-template.yml@main
```

Comment:
Need to add the inputs to the workflow

Reply:
Fixed in [870d3cb](https://github.com/ballerina-platform/module-ballerinax-oracledb/pull/604/commits/870d3cb783e845ca398238a8e2358ed0814f14f8)